examples/prisma-react/src/TodoList.tsx

```diff
@@ -0,0 +1,60 @@
+// @ts-ignore
+import { Todo } from '@prisma/client'
+import { useEffect, useState } from 'react'
+import NewTodo from './NewTodo'
+import { onDeleteTodo, onGetTodos, onToggleTodo } from './TodoList.telefunc'
+
+export { TodoList }
+
+function TodoItem({ refetch, ...todo }: Todo & { refetch: () => void }) {
+  return (
+    <li key={todo.id}>
+      <h2>
+        {todo.title}{' '}
+        <button
+          id={'toggle-' + todo.title}
```

Comment:
~~Shouldn't it be `todo.id`?~~ Do we need an `id` here?

Reply:
Got it. It should then be `todo.id` then though. That's the purpose of an ID after all ;-).